_sass/pages/_provider-map.scss

```diff
@@ -0,0 +1,52 @@
+$map-height-sm: 460px;
+$map-height-md: 560px;
+
```

Comment:
Can we get a little extra space below the intro content please? Something like (this is probably a horrible rule, but you get the idea):

```sass
.provider-map .main-primary > .container:first-of-type {
    padding-bottom: 1rem
}
```